defaults/main.yml

```
@@ -39,6 +39,7 @@ monitoring_grafana_enabled: true
 monitoring_alertmanager_enabled: true
 monitoring_blackbox_enabled: true
 monitoring_cadvisor_enabled: true
+monitoring_postgresql_enabled: true
```

```suggestion
monitoring_pgsql_exporter_enabled: true
```
The names of the variables should be consistent. Check all the variables and make sure that they have similar prefixes, for example `monitoring_pgsql_exporter_`